File: Android/sources/com/android/billingclient/api/zzaq.java
package com.android.billingclient.api;

import java.util.concurrent.Callable;

final class zzaq implements Callable {
    final /* synthetic */ String zza;
    final /* synthetic */ PurchasesResponseListener zzb;
    final /* synthetic */ BillingClientImpl zzc;

    public zzaq(BillingClientImpl billingClientImpl, String str, PurchasesResponseListener purchasesResponseListener) {
        this.zza = str;
        this.zzb = purchasesResponseListener;
        this.zzc = billingClientImpl;
    }

    public final Object call() throws Exception {
        zzcx zzaf = BillingClientImpl.zzaf(this.zzc, this.zza, 9);
        if (zzaf.zzb() != null) {
            this.zzb.onQueryPurchasesResponse(zzaf.zza(), zzaf.zzb());
            return null;
        }
        PurchasesResponseListener purchasesResponseListener = this.zzb;
        BillingResult zza2 = zzaf.zza();
        nxg nxg = rxg.b;
        purchasesResponseListener.onQueryPurchasesResponse(zza2, ayg.v);
        return null;
    }
}

File: IOS/v2/com/android/billingclient/api/zzaq.swift
package com.iOS.billingclient.api;.swift
import java.util.concurrent.Callable;.swift
final class zzaq implements Callable {.swift
final /* synthetic */ String zza;.swift
final /* synthetic */ PurchasesResponseListener zzb;.swift
final /* synthetic */ BillingClientImpl zzc;.swift
public zzaq(BillingClientImpl billingClientImpl, String str, PurchasesResponseListener purchasesResponseListener) {.swift
this.zza = str;.swift
this.zzb = purchasesResponseListener;.swift
this.zzc = billingClientImpl;.swift
}.swift
public final Object call() throws Exception {.swift
zzcx zzaf = BillingClientImpl.zzaf(this.zzc, this.zza, 9);.swift
if (zzaf.zzb() = null) {.swift
this.zzb.onQueryPurchasesResponse(zzaf.zza(), zzaf.zzb());.swift
return null;.swift
}.swift
PurchasesResponseListener purchasesResponseListener = this.zzb;.swift
BillingResult zza2 = zzaf.zza();.swift
nxg nxg = rxg.b;.swift
purchasesResponseListener.onQueryPurchasesResponse(zza2, ayg.v);.swift
return null;.swift
}.swift
}.swift
